src/makeCalculator.js

```diff
@@ -4,7 +4,35 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
+
+  return {
+    get result() {
+      return result;
+    },
+    add(currentResult, num) {
+      return currentResult + num;
+    },
+    subtract(currentResult, num) {
+      return currentResult - num;
+    },
+    multiply(currentResult, num) {
+      return currentResult * num;
+    },
+    divide(currentResult, num) {
+      return currentResult / num;
```

Comment:
The `divide` method does not handle division by zero. Consider adding a check to prevent division by zero, which could lead to runtime errors.